packages/cms-base/helpers/html-to-vue/getOptionsFromNode.ts

```diff
@@ -1,15 +1,41 @@
 type Options = {
-  style: String;
-  class: String;
-  attrs: Object;
+  align: String | null;
+  attrs: Object | null;
+  class: String | null;
+  style: String | null;
```

Comment:
```suggestion
  align?: string;
  attrs?: Record<string, string>;
  class?: string;
  style?: string;
```
wouldn't be better to have it as undefined if not existing?